docker/readmodel-db/delegation.sql

```diff
@@ -0,0 +1,37 @@
+CREATE SCHEMA IF NOT EXISTS readmodel_delegation;
+
+CREATE TABLE IF NOT EXISTS readmodel_delegation.delegation(
+  id UUID,
+  metadata_version INTEGER NOT NULL,
+  delegator_id UUID NOT NULL,
+  delegate_id UUID NOT NULL,
+  eservice_id UUID NOT NULL,
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL,
+  updated_at TIMESTAMP WITH TIME ZONE,
+  rejection_reason VARCHAR,
+  state VARCHAR NOT NULL,
+  kind VARCHAR NOT NULL,
+  PRIMARY KEY (id)
+);
+
+CREATE TABLE IF NOT EXISTS readmodel_delegation.delegation_stamp(
+  delegation_id uuid NOT NULL REFERENCES readmodel_delegation.delegation(id) ON DELETE CASCADE,
+  metadata_version integer NOT NULL,
+  who uuid NOT NULL,
+  "when" timestamp WITH time zone NOT NULL,
+  kind varchar NOT NULL,
+  PRIMARY KEY (delegation_id, kind)
+);
+
+CREATE TABLE IF NOT EXISTS readmodel_delegation.delegation_contract_document(
+  id UUID,
+  delegation_id UUID NOT NULL REFERENCES readmodel_delegation.delegation (id) ON DELETE CASCADE,
+  metadata_version INTEGER NOT NULL,
+  name VARCHAR NOT NULL,
+  content_type VARCHAR NOT NULL,
+  pretty_name VARCHAR NOT NULL,
+  path VARCHAR NOT NULL,
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL,
+  kind VARCHAR NOT NULL,
```

Comment:
Same comment left for agreement contract/eservice interface: in this way a delegation could have more contracts of the same kind, I would add contract ids inside the delegation table